modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Sequence.scala

```diff
@@ -342,6 +348,10 @@ object Sequence {
 
     }
 
+    object Zipper {
+      implicit val eq: Equal[Zipper] = Equal.equalA
```

Comment:
I assume this is just a stub. You should't use `.equalA` unless you have a type that you know for sure has a reasonable universal equality.

Reply:
I assume `case classes` have a good universal equality

Reply:
Is that a fair assumption? if a case class is composed of case classes it should have a correct `equals` right?

Reply:
It's only true if the fields have reasonable equality; i.e., `Int`, `String`, stuff like that; or are products or sums thereof. Eventually it all gets down to universal equality on primitive types.